relayer/relayer.go

```diff
@@ -187,16 +187,6 @@ func NewRelayer(
 
 // RelayMessage relays a single warp message to the destination chain. Warp message relay requests from the same origin chain are processed serially
 func (r *Relayer) RelayMessage(warpLogInfo *vmtypes.WarpLogInfo, metrics *MessageRelayerMetrics, messageCreator message.Creator) error {
-	// Check that the destination chain ID is supported
-	if !r.CheckSupportedDestination(warpLogInfo.DestinationChainID) {
```

Comment:
We should keep this in. If we extract the destination chain ID from `messageManager` in this function, we can then call `CheckSupportedDestination` with that value.